### ISSUE TITLE
a11y P3 - Related Content on PGL pages shouldn't have complimentary region

### ISSUE THREAD

**Issue by HarveyPeachey** [bbc/simorgh#5516]
**Describe the bug**

The related content component at the bottom of the page - Should not be complimentary region - This should be an HTML section with a role of 'region' that is labelled by the components h2. 

Note this is just for this asset type, other asset types may have different requirements such as some of the Media pages. Would be good to check that this is correct elsewhere we are using this too. Discuss with Media team if they want this to change for Media pages or stay the same.

**To Reproduce**
Steps to reproduce the behaviour:

1. Navigate to Related Content with any Screen Reader.

**Expected behaviour**
It should be an HTML section with a role of 'region' that is labelled by the components h2. 

**Screenshots**

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
`<aside>` take a look at the MDN documentation to see what the behaviour of `role` has: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside.

**Comment by andrewscfc** [comment]
@jimjohnsonrollings @lizcameron @aeroplanejane  Natasha

Just looking at this issue and we need to adjust the type of landmark for this component for PGLs as the content here is not related to the main content e.g. not complementary. We have agreed this is the case for PGLS but what about:

- STYs
- MAPs

Ideally from a technical perspective it would be easier to use the same markup everywhere but we can do a more complex solution if sometimes the 'Related Content' is not _actually_ related to the main body content (that does sound a bit of a strange situation to me?)

**Comment by aeroplanejane** [comment]
@andrewscfc In the case of MAPs related content will always be related to the main body content, so this sounds good to me

**Comment by jimjohnsonrollings** [comment]
For STYs and PGLs I think "region" is fine.  As Catharine and I discussed yesterday, we don't think the content is sufficiently related to use complementary region.

**Comment by aeroplanejane** [comment]
We're happy if same is applied to our pages. See-alsos / related content is has the same application on MAPs. Thanks!
